sentinelhub/exceptions.py

```diff
@@ -45,6 +45,28 @@ class SHRuntimeWarning(RuntimeWarning):
     """
 
 
+def handle_deprecated_data_source(data_collection, data_source, default=None):
```

Comment:
I like the approach, but I'd suggest putting this to data_collections.py.

Reply:
You are right. I moved it to data_collections.